blocks/timeline/index.js

```diff
@@ -171,33 +129,16 @@ registerBlockType( 'rtgb/timeline', {
 
 					<div className="content-wrap">
 						<div className="content-inner">
-							<RichText
-								className={ 'timeline-title' }
-								tagName={ 'h3' }
-								onChange={ onTitleChange }
-								value={ timelineTitle }
-								focus={ setFocus }
-								placeholder={ __( 'Title' ) }
-								onFocus={ onFocusTitle }
-							/>
-
-							<RichText
-								tagName={ 'p' }
-								className={ 'timeline-description' }
-								placeholder={ __( 'Enter contributors list here' ) }
-								value={ timelineContent }
-								onChange={ onContentChange }
-								focus={ setFocus }
-								onFocus={ onFocusContent }
-							/>
+							{ middleware.fields.title }
+							{ middleware.fields.content }
 
 							<div className="postlinks">
-								{ ( newslink || bloglink ) ? <strong>Links: </strong> : null }
+								{ ( newsLink || blogLink ) ? <strong>Links: </strong> : null }
```

Comment:
"Links" should be in translation function `__()`

Reply:
Done